### PR TITLE
[Python API] Remove old api class from the new api

### PR DESCRIPTION
**Pull request by akuporos**
### Details:
 - old API class (InferenceEngine::Parameter) file removal from pyopenvino
### Ticket:
 - 81339

### CODE REVIEW

**Review by jiwaszki — APPROVED** [review]
Overall LGTM, please fix CIs issues with imports.

**Review by AlexeyLebedev1 — APPROVED** [review]

**Review by ilyachur — APPROVED** [review]

**Comment by ilya-lavrenov** [comment]
Let's make CI green

File: src/bindings/python/src/pyopenvino/utils/utils.cpp [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+            PyDict_SetItemString(dict, property_name.c_str(), PyUnicode_FromString(mutability.c_str()));
+        }
+        return py::cast<py::object>(dict);
+    } else {
+        PyErr_SetString(PyExc_TypeError, "Failed to convert parameter to Python representation!");
+        return py::cast<py::object>((PyObject*)NULL);
+    }
+}
+};  // namespace utils
+};  // namespace Common
```
Comment (AlexeyLebedev1) [inline src/bindings/python/src/pyopenvino/utils/utils.cpp:155]: empty line.

File: src/bindings/python/src/pyopenvino/utils/utils.cpp [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    // Check for std::tuple<unsigned int, unsigned int>
+    else if (any.is<std::tuple<unsigned int, unsigned int>>()) {
+        auto val = any.as<std::tuple<unsigned int, unsigned int>>();
+        PyObject* tuple = PyTuple_New(2);
+        PyTuple_SetItem(tuple, 0, PyLong_FromUnsignedLong((unsigned long)std::get<0>(val)));
+        PyTuple_SetItem(tuple, 1, PyLong_FromUnsignedLong((unsigned long)std::get<1>(val)));
+        return py::cast<py::object>(tuple);
+    }
+    // Check for std::tuple<unsigned int, unsigned int, unsigned int>
+    else if (any.is<std::tuple<unsigned int, unsigned int, unsigned int>>()) {
```
Comment (AlexeyLebedev1) [inline src/bindings/python/src/pyopenvino/utils/utils.cpp:113]: Is it covered by tests? I see only test for tuple with two elements.

Comment (akuporos) [reply src/bindings/python/src/pyopenvino/utils/utils.cpp:113]: I'll cover it by another PR.

File: src/bindings/python/src/pyopenvino/utils/utils.cpp (hunk 1 above)
Comment (akuporos) [reply src/bindings/python/src/pyopenvino/utils/utils.cpp:155]: I'll fix it by another PR.

Comment (akuporos) [reply src/bindings/python/src/pyopenvino/utils/utils.cpp:155]: 83555 ticket

**Review by jiwaszki — APPROVED** [review]

**Review by AlexeyLebedev1 — APPROVED** [review]

**Pull request merged**